localization/simp_chinese/newsfeed_suppress_l_simp_chinese.yml

```
@@ -0,0 +1,5 @@
+﻿l_simp_chinese:
+ newsfeed_suppress: "信息摘要_折叠"
```

This should actually stay `"newsfeed_suppress"` regardless of localization - it's used internally and not meant to be shown to the player.

```suggestion
 newsfeed_suppress: "newsfeed_suppress"
```